### ISSUE TITLE
10.0.17763.2300 not supported

### ISSUE THREAD

**Issue by zzelalex** [stascorp/rdpwrap#1596]
The question.
And the answer. Enjoy.
```

[10.0.17763.2300]
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=7E311
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x64=1
SingleUserOffset.x64=13879
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=18435
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
SLInitHook.x64=1
SLInitOffset.x64=2179C
SLInitFunc.x64=New_CSLQuery_Initialize

[10.0.17763.2300-SLInit]
bInitialized.x64      =F3AC4
bServerSku.x64        =F3AC8
lMaxUserSessions.x64  =F3ACC
bAppServerAllowed.x64 =F3AD4
bRemoteConnAllowed.x64=F3AD8
bMultimonAllowed.x64  =F3ADC
ulMaxDebugSessions.x64=F3AE0
bFUSEnabled.x64       =F3AE4

```

**Comment by dxdiv** [comment]
not work

**Comment by zzelalex** [comment]
> not work

Keep trying

![Capture](https://user-images.githubusercontent.com/43055218/141132189-f234bc8a-43b9-4364-bdfd-31ecb3925f8f.PNG)

**Comment by dxdiv** [comment]
Yes, installed, running, listening and fully support, but "Another user is signed in. If you continue, they’ll be disconnected. Do you want to sign in anyway?"... crap. Was forced to delete KB5007206.

**Comment by sebaxakerhtc** [comment]
> 
> 
> The question. And the answer. Enjoy.
> 
> ```
> 
> [10.0.17763.2300]
> LocalOnlyPatch.x64=1
> LocalOnlyOffset.x64=7E311
> LocalOnlyCode.x64=jmpshort
> SingleUserPatch.x64=1
> SingleUserOffset.x64=13879
> SingleUserCode.x64=Zero
> DefPolicyPatch.x64=1
> DefPolicyOffset.x64=18435
> DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
> SLInitHook.x64=1
> SLInitOffset.x64=2179C
> SLInitFunc.x64=New_CSLQuery_Initialize
> 
> [10.0.17763.2300-SLInit]
> bInitialized.x64      =F3AC4
> bServerSku.x64        =F3AC8
> lMaxUserSessions.x64  =F3ACC
> bAppServerAllowed.x64 =F3AD4
> bRemoteConnAllowed.x64=F3AD8
> bMultimonAllowed.x64  =F3ADC
> ulMaxDebugSessions.x64=F3AE0
> bFUSEnabled.x64       =F3AE4
> ```

Attach your termsrv.dll

**Comment by faaring** [comment]
> The question. And the answer. Enjoy.
> 
> ```
> 
> [10.0.17763.2300]
> LocalOnlyPatch.x64=1
> LocalOnlyOffset.x64=7E311
> LocalOnlyCode.x64=jmpshort
> SingleUserPatch.x64=1
> SingleUserOffset.x64=13879
> SingleUserCode.x64=Zero
> DefPolicyPatch.x64=1
> DefPolicyOffset.x64=18435
> DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
> SLInitHook.x64=1
> SLInitOffset.x64=2179C
> SLInitFunc.x64=New_CSLQuery_Initialize
> 
> [10.0.17763.2300-SLInit]
> bInitialized.x64      =F3AC4
> bServerSku.x64        =F3AC8
> lMaxUserSessions.x64  =F3ACC
> bAppServerAllowed.x64 =F3AD4
> bRemoteConnAllowed.x64=F3AD8
> bMultimonAllowed.x64  =F3ADC
> ulMaxDebugSessions.x64=F3AE0
> bFUSEnabled.x64       =F3AE4
> ```

Thanks! It didn't work the first time. But when added to the beginning of the file, it worked.

**Comment by faaring** [comment]
> not work

Works. The last line in the file must exist and be empty.

**Comment by faaring** [comment]
![image](https://user-images.githubusercontent.com/11716838/141236654-38f1fcfa-3417-4111-8902-bebb6c691948.png)

**Comment by Raad1n** [comment]
it helped
1. rdpwinst.exe - u
2. rdpwinst.exe - i
3. autoupdate

**Comment by trogper** [comment]
> > not work
> 
> Works. The last line in the file must exist and be empty.

I have just burned on this AFTER reading your comment 🤦‍♂️

**Comment by kappo96** [comment]
it works，after restart your computer

**Comment by trogper** [comment]
No need to restart the whole computer. Just restart the RDP service
